Cargo.toml

```diff
@@ -12,11 +12,17 @@ repository = "https://github.com/m4b/capstone-sys"
 libc = "0.2"
 
 [build-dependencies]
+bindgen = "0.26.3"
```

Comment:
This can be optional with undeleted bindings; just need to add a feature flag on the bindings and reexport iiuc